### ISSUE TITLE
I got rick rolled

### ISSUE THREAD

**Issue by agausmann** [dnorhoj/AntiRickRoll#32]
https://www.youtube.com/watch?v=KPJDlgPA3TA
https://tinyurl.com/5x2bcwjy

**Comment by IdkWhatToCallMe123** [comment]
I'm just so sad to hear that you had to suffer through this

**Comment by ingoau** [comment]
I think at some point they will bring in rickrolldb were you can contribute and add that url. I am working on a project that can scan youtube videos for rickrolls

**Comment by ingoau** [comment]
You may want to create a pr like this one:
![image](https://github.com/dnorhoj/AntiRickRoll/assets/96573515/ec6c9329-c602-43da-8c20-c957dfda6b44)
You would want to fork and then go to /src/background/blockedIds.js 
Then add it and create a pr

**Comment by agausmann** [comment]
@Inglan Yes I usually would, but PRs for lists like this end up making a ton of merge conflicts (as evidenced in your screenshot) that the PR authors or the maintainers have to resolve, because everyone's appending to the same location in the file, at the end of the list.
It's just much easier for the maintainer to make those commits directly.

**Comment by superintendent2521** [comment]
currently working on a fork with the db working
